utility/importer/chrome_importer.cc

```diff
@@ -86,11 +86,14 @@ void ChromeImporter::StartImport(const importer::SourceProfile& source_profile,
     bridge_->NotifyItemEnded(importer::PASSWORDS);
   }
 
+// issues/7044: We do not import cookies from Chrome.
+#if 0
```

Comment:
if we remove the cookies import code for Firefox, there is no point keeping the code for Chrome

Reply:
ok, removed and left a comment

Reply:
done